### PR TITLE
add Oak Network to parallel auction toolkit

### PR DESCRIPTION
**Pull request by justinzhou93**
## Onboarding issue link

* Before request the review, please open an issue based on the [onBoarding template](./ISSUE_TEMPLATE/onborading.md);
* Provide link to the onBoarding issue: https://github.com/parallel-finance/auction-toolkit/issues/193

### CODE REVIEW

File: crowdloans/projects/polkadot.json [hunk 1]
```diff
@@ -704,5 +704,29 @@
     "icon": "https://auction-us-east-2.s3.us-east-2.amazonaws.com/icons/chains/aventus.png",
     "website": "https://www.aventus.io/",
     "overview": "Aventus is an enterprise-grade blockchain that marries the scale of a permissioned blockchain with the security and interoperability of public blockchains — with none of the drawbacks of either world.Founded in 2016, Aventus makes building on Polkadot, Ethereum and beyond accessible for businesses and decentralised platforms, bringing interoperability, scalability and sustainability to Ethereum transactions. In its first year of operation, the Aventus Network has onboarded over 20 million transactions for business clients with 2 million active wallets.Aventus’ mission is to enable companies in any ecosystem to capitalise on a vast series of use cases – including NFTs, video games, loyalty schemes, event ticketing, supply chain provenance, and cargo management – via enterprise-grade modular architecture and future-proof themselves via blockchain interoperability."
+  },
+  {
+    "paraid": "2090",
```
Comment (irsal) [inline crowdloans/projects/polkadot.json:709]: this is not locked in yet

File: crowdloans/projects/polkadot.json [hunk 2]
```diff
@@ -704,5 +704,29 @@
...
+    "name": "Oak Network",
```
Comment (irsal) [inline crowdloans/projects/polkadot.json:710]: `OAK Network`

File: crowdloans/projects/polkadot.json [hunk 3]
```diff
@@ -704,5 +704,29 @@
...
+    "token": "OAK",
+    "description": "Automation Blockchain for Polkadot",
+    "overview": "Oak is an automation blockchain for Defi and recurring payments across protocols. Oak aims to be the best-in-class automation utility across protocols, offering on-chain automation not possible today.",
```
Comment (irsal) [inline crowdloans/projects/polkadot.json:713]: `OAK delivers trustless automation, dedicating its block space and optimizing gas to store, trigger, and execute transactions on connected blockchains.`

File: crowdloans/rewards/polkadot.json [hunk 4]
```diff
@@ -2175,5 +2175,31 @@
...
+      {
+        "label": "Total Rewards",
+        "content": ["Up to 150,000,000 OAK (15% of total token supply)"]
+      },
+      {
+        "label": "Bonus",
+        "content": [
+          "Crowdloan Opening Bonus: Contributors who contribute before 22 Sept 2022 18:00 UTC get extra 10%",
+          "Crowdloan Ending Bonus: Contributors who contribute after 22 Sept 2022 18:00 UTC get extra 5%",
+          "Referral: Reward 10% of receivers' base split between sender and receiver",
```
Comment (irsal) [inline crowdloans/rewards/polkadot.json:2193]: technically referral won't be available through the parallel app

File: crowdloans/rewards/polkadot.json [hunk 5]
```diff
@@ -2175,5 +2175,31 @@
...
+    "reward": {
+      "min": 75.5
+    },
+    "rewardRules": [
+      {
+        "label": "Total Rewards",
+        "content": ["Up to 150,000,000 OAK (15% of total token supply)"]
+      },
+      {
+        "label": "Bonus",
```
Comment (irsal) [inline crowdloans/rewards/polkadot.json:2189]: I'd add 4 PARA and cDot, see: https://oak.tech/oak/reward/#base-reward

File: crowdloans/rewards/polkadot.json [hunk 6]
```diff
@@ -2175,5 +2175,31 @@
...
+        "content": [
+          "Crowdloan Opening Bonus: Contributors who contribute before 22 Sept 2022 18:00 UTC get extra 10%",
+          "Crowdloan Ending Bonus: Contributors who contribute after 22 Sept 2022 18:00 UTC get extra 5%",
+          "Referral: Reward 10% of receivers' base split between sender and receiver",
+          "Community Bonus: OAK crowdloan tokens earned through group effort of all participants who contributed above 5 DOT. Every participant who contributes a minimum of 5 DOT will receive a community bonus based on the total number of contributors. This reward is to encourage decentralization and incentivize contributors to grow the community."
+        ]
+      },
+      {
+        "label": "Vesting schedule",
```
Comment (irsal) [inline crowdloans/rewards/polkadot.json:2198]: is this specifically for the crowdloan rewards?

**Review by irsal — CHANGES_REQUESTED** [review]

File: crowdloans/projects/polkadot.json (hunk 1 above)
Comment (justinzhou93) [reply crowdloans/projects/polkadot.json:709]: should be locked now: 2090

File: crowdloans/rewards/polkadot.json [hunk 7]
```diff
@@ -2175,5 +2175,31 @@
...
     ]
+  },
+  "2090": {
+    "reward": {
+      "min": 75.5
+    },
+    "rewardRules": [
+      {
+        "label": "Total Rewards",
+        "content": ["Up to 150,000,000 OAK (15% of total token supply)"]
```
Comment (irsal) [inline crowdloans/rewards/polkadot.json:2186]: 1.5%

File: crowdloans/rewards/polkadot.json [hunk 8]
```diff
@@ -2175,5 +2175,31 @@
...
+      },
+      {
+        "label": "Bonus",
+        "content": [
+          "Crowdloan Opening Bonus: Contributors who contribute before 22 Sept 2022 18:00 UTC get extra 10%",
+          "Crowdloan Ending Bonus: Contributors who contribute after 22 Sept 2022 18:00 UTC get extra 5%",
+          "Community Bonus: OAK crowdloan tokens earned through group effort of all participants who contributed above 5 DOT. Every participant who contributes a minimum of 5 DOT will receive a community bonus based on the total number of contributors. This reward is to encourage decentralization and incentivize contributors to grow the community.",
+          "Parallel Bonus: 4 PARA and 1 cDot."
```
Comment (irsal) [inline crowdloans/rewards/polkadot.json:2194]: per DOT contributed

**Review by irsal — CHANGES_REQUESTED** [review]
more feedback

**Review by irsal — APPROVED** [review]
lgtm from the oak side

**Review by crystalynliu — APPROVED** [review]
LGTM

**Comment by justinzhou93** [comment]
hey @crystalynliu , sorry but would you mind re-lgtm'ing again? there was a merge conflict

**Review by crystalynliu — APPROVED** [review]

**Pull request merged**